schemas/+behavior/TowersBlockTrialVideo.m

```diff
@@ -0,0 +1,15 @@
+%{
+-> behavior.TowersBlockTrial
+---
+video_path:                 varchar(255)         # video directory + filename for each trial
+%}
+
+classdef TowersBlockTrialVideo < dj.Imported
+   % properties(SetAccess=protected)
+   %     master = behavior.TowersBlock
+   % end
+   methods(Access=protected)
+        function makeTuples(self, key)
+        end
+   end
+end
```

Comment:
You'll specify the makeTuples later right?

Reply:
The idea is that the insertion of this table is controlled by a class that generates trial videos.

It would be called like this:
key.subject_fullname = 'hnieh_e84'
key.session_date = '2019-06-07'
generateVideosSession(key)

Inside the generateVideosSession object I got an insertion to this table:
insert(behavior.TowersBlockTrialVideo, trialKey);